src/snowflake/snowpark/types.py

```diff
@@ -458,10 +460,10 @@ class PandasDataFrameType(_PandasType):
     """
 
     def __init__(
-        self, col_types: Iterable[DataType], col_names: Iterable[str] = None
```

Comment:
I wanna make this change since the documentation says the input is a List. https://docs.snowflake.com/en/developer-guide/snowpark/reference/python/latest/api/snowflake.snowpark.types.PandasDataFrameType